shared/constants/swaps.ts

```diff
@@ -175,14 +175,14 @@ export const GAS_API_BASE_URL = 'https://gas.api.cx.metamask.io';
 export const GAS_DEV_API_BASE_URL = 'https://gas.uat-api.cx.metamask.io';
 
 const BSC_DEFAULT_BLOCK_EXPLORER_URL = 'https://bscscan.com/';
-const MAINNET_DEFAULT_BLOCK_EXPLORER_URL = 'https://etherscan.io/';
+export const MAINNET_DEFAULT_BLOCK_EXPLORER_URL = 'https://etherscan.io/';
```

Comment:
I don't think we need this; I think we can use `BUILT_IN_NETWORKS[NETWORK_TYPES.MAINNET].blockExplorerUrl`